.github/workflows/check-merge-conflict.yml

```
@@ -0,0 +1,16 @@
+name: Merge Conflict Check
+on:
+  push:
+  pull_request_target:
+    types: [synchronize]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: check if prs are dirty
```

```suggestion
      - name: Check for dirty PRs
```

I would prefer this phrasing, but that's not the point here. The fact that the workflow name has proper casing, but this doesn't triggers me. :see_no_evil:

---

The, let's go for
```suggestion
      - name: check for dirty pull requests
```

---

But that's still cased inconsistently with the workflow `name`! Where is my `:triggered:` emoji?!

---

I hope it's fine like this.